node.py

```diff
@@ -0,0 +1,5 @@
+class Node:
```

Comment:
The `node.py` `player.py` and `game.py` have a few very light wrapper data structures that you could consider putting in to a module (or file) called `model` (or `model.py`). The *model* here means the *data model* for your program; it's the set of abstractions / entities that exist in the world of the program. 

In a database application like Django, each model = a table in the database where each row in that table corresponds to an instance of the model class (e.g. Cars table has a `Car` class, and any `car1 = Car()` would have a row in the database for `car1`.

While this change isn't very substantive, the shift of moving these kinds of entities into a "model" location helps landmark the code for other programmers to understand